src/build.xml

```diff
@@ -5,7 +5,8 @@
 <PropertyGroup>
     <Solution>Dynamo.All.sln</Solution>
     <Platform>Any CPU</Platform>
-    <RuntimeIdentifier>win10-x64</RuntimeIdentifier>
+    <UseDotNet>net6.0</UseDotNet>
```

Comment:
https://github.com/DynamoDS/Dynamo/pull/14618/files#diff-bbde35c3169d8580248bcea0e6c8e5722086cba412a65c737fb4c8656d095c55R9
I added a new property in our .props files so that we can set the TargetFramework from the commandline without forcing it on projects that already override it (like DynamoServices - netstandard2.0)

Reply:
Oops, there is supposed to be only one. UseDotNet should be DotNet
I will change it